CONTRIBUTING_NI.md

```diff
@@ -32,7 +32,7 @@ git describe --tags --abbrev=0 main
 git checkout -b ni/pub/cool-new-feature <latest-tag>
 ```
 
-Then make your changes on this branch and open a pull request merging it into _main_. If your changes on an ni/pub/* branch result in merge conflicts with another ni/pub/* branch, rebase your branch on the conflicting branch and force push. After the changes are reviewed and accepted, the branch should be merged **via a merge commit** (not squashing) and **not** deleted. Merge commits will make it easier to undo the changes later on if they are accepted upstream.
+Then make your changes on this branch and open a pull request merging it into _main_. If your changes on an ni/pub/* branch result in merge conflicts with another ni/pub/* branch, rebase your branch on the conflicting branch and force push. After the changes are reviewed and accepted, the branch should be **squash merged** and **not** deleted.
```

Comment:
Trying to maintain merge commits ended up being too difficult when we did the last rebase, so I decided to abandon this idea.

Reply:
I'm guessing this is why the documentation wasn't updated last time. I recall it being a pain and we mentioned we needed to revisit this decision.